tests/magick-image/add-profile.spec.ts

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright Dirk Lemstra https://github.com/dlemstra/magick-wasm.
+// Licensed under the Apache License, Version 2.0.
+
+import { IMagickImage, MagickImage } from '../../src/magick-image';
+import { TestImages } from '../test-images';
+
+let image: IMagickImage;
+
+beforeEach(() => {
+    image = MagickImage.create();
+});
+
+afterEach(() => {
+    image.dispose();
+});
+
+describe('MagickImage#addProfile', () => {
+    it('should add the profile', () => {
+
```

Comment:
Can you again move this empty line. The rest of the empty lines can stay because they are in style with the rest of the code.